src/Tribe/Linked_Posts.php

```diff
@@ -415,6 +424,11 @@ public function get_linked_post_types() {
 	 * @return array
 	 */
 	public function get_linked_post_info( $linked_post_type, $args = array(), $linked_post_ids = null ) {
+		$cache_key = $this->cache->make_key( func_get_args(), 'linked_post_info_' );
```

Comment:
You cannot use `func_get_args()` directly into a function/method call, you need to save it in a variable first. PHP 5.2 compat.